src/highrl/utils/calculations.py

```diff
@@ -79,78 +56,13 @@ def point_to_obstacle_distance(
     return distances
 
 
-def point_to_point_distance(
-    first_point: Tuple[int, int],
-    second_point: Tuple[int, int],
-) -> float:
-    """Get distance between two points
-
-    Args:
-        first_point (Tuple[int, int]): First points coordinates
-        second_point (Tuple[int, int]): Second points coordinates
-
-    Returns:
-        float: Distance between points
-    """
-    point1_x, point1_y = first_point
-    point2_x, point2_y = second_point
-
-    return ((point1_x - point2_x) ** 2 + (point1_y - point2_y) ** 2) ** 0.5
-
-
-def cross_product_point_line(
-    point: List[int], p_first: List[int], p_sec: List[int]
-) -> Union[float, int]:
-    """Calculate the cross product between a point and a line
-
-    Args:
-        point (List[int]): input point
-        p_first (List[int]): first point of the line
-        p_sec (List[int]): second point of the line
-
-    Returns:
-        float: value of the cross product
-    """
-    ground_point: np.ndarray = np.array(point, dtype=np.float32)
-    line_point1: np.ndarray = np.array(p_first, dtype=np.float32)
-    line_point2: np.ndarray = np.array(p_sec, dtype=np.float32)
-    left_point = ground_point - line_point1
-    right_point = ground_point - line_point2
-    cross_product = left_point[0] * right_point[1] - left_point[1] * right_point[0]
-    return cross_product
-
-
-def cross_product_triangle(
-    point: Tuple[int, int],
-    p_first: Tuple[int, int],
-    p_sec: Tuple[int, int],
-) -> Union[float, int]:
-    """Triagular cross product
-    (p1 - p) x (p2 - p)
-    Args:
-        point (List[int]): input point
-        p_first (List[int]): first point of the line
-        p_sec (List[int]): second point of the line
-
-    Returns:
-        float: triangular cross product value
-    """
-    ground_point = np.array(point, dtype=np.float32)
-    pnt1 = np.array(p_first, dtype=np.float32)
-    pnt2 = np.array(p_sec, dtype=np.float32)
-    return np.cross((pnt1 - ground_point), (pnt2 - ground_point)).tolist()
-
 def neg_exp(base: float, exponent: float) -> float:
     """Exponentiation for negative numbers
 
-    Args:
-        base (float): Base of exponent
-        exponent (float): Exponent
-
-    Returns:
-        float: sign * (|base|**exponont)
+    Since exponentiation of negative numbers might result
+    in complex numbers, we omit this by returning sign * |base| ** exponent
```

Comment:
Why did you delete the following part of the doc-string?

  Args:
        base (float): Base of exponent
        exponent (float): Exponent
    Returns:
        float: sign * (|base|**exponont)

Reply:
It is redundant, as these details can already be inferred from the type hints and the new doc-string.